src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerEnums.java

```diff
@@ -0,0 +1,26 @@
+package com.minecolonies.coremod.entity.ai.citizen.baker;
+
+/**
+ * All enums the baker needs.
+ */
+public final class BakerEnums
+{
+    /**
+     * Private constructor to hide the implicit one.
+     */
+    private BakerEnums()
```

Comment:
Unless you think we are going to have more BakerEnums, just remove this wrapper class.